www/IndoorAtlas.js

```diff
@@ -507,7 +507,40 @@ var Wayfinder = function(wayfinderId) {
    */
   this.getRoute = function() {
     return new Promise(function(resolve, reject) {
-      var success = function(result) { resolve(result) };
+      var success = function(result) {
+
+        var arrayOfRoutes = [];
+        result.route.map(function(route) {
+
+          var beginNodeIndex;
+          var endNodeIndex;
+          var edgeIndex;
+
+          if (!route.begin.nodeIndex) {
```

Comment:
There should be no need to check for these checks/conversions here (either do the check in the RoutingPoint constructor or not at all).

And is the point changing `undefined` to `null` since the API docs say it's `null`? If so, it should be done explicitly by checking `if (x === undefined) x = null`, since `!x` is also true for, e.g. `x = 0`. This was also the problem in #18.